tests/dummy/app/templates/tip.hbs

```diff
@@ -176,3 +193,300 @@
     odio tortor, ac fringilla odio dictum a. Pellentesque molestie lobortis nisi a faucibus. Donec at sapien sed dui malesuada aliquet. Donec neque felis, faucibus vitae lorem sit amet, aliquet vulputate diam. Fusce ut auctor ipsum. Proin porttitor dapibus eros vel placerat. Duis id dui euismod, euismod ante ut, mattis urna. Quisque semper eros nec turpis adipiscing tempus vel sit amet quam. Fusce vestibulum urna ac eros commodo, vitae tristique metus luctus. Etiam sit amet sollicitudin purus. Maecenas nec rhoncus metus. Cras sit amet tempor eros, at cursus est. Nullam pellentesque orci et tellus luctus cursus. Nullam pulvinar fermentum ante, a feugiat mauris fringilla quis.</p>
   </div>
 </div>
+<div class="spacer">
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p><strong><em>scroll to the next example</em></strong></p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
+  <p>&middot;</p>
```

Comment:
I did it in the first section to force the page to be tall enough to scroll (while being obvious that there is more stuff below the fold). Now it's more or less a fancy divider.